### PR TITLE
Berlin landing page variations

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description
Three /new variations for the Berlin campaign (speed, privacy, oys).
- Set up base file for similar variations (based on portland campaign)
- Add 3 variations 
  - assets
  - styles
  - copy
  - tests

## Issue / Bugzilla link
[Fix bug 1473357](https://bugzilla.mozilla.org/show_bug.cgi?id=1473357)
Closes https://github.com/mozilla/bedrock/issues/5880

## Testing
Pay extra attention to the different break points please.

https://bedrock-demo-shobson.oregon-b.moz.works/de/firefox/new/?xv=geschwindigkeit
https://bedrock-demo-shobson.oregon-b.moz.works/de/firefox/new/?xv=privatsphare
https://bedrock-demo-shobson.oregon-b.moz.works/de/firefox/new/?xv=auf-deiner-seite

### CODE REVIEW

**Comment by stephaniehobson** [comment]
Ready for review.

Waiting on the client for slug names but we can merge now and rename later.

**Comment by stephaniehobson** [comment]
Slugs have final names now.

**Comment by stephaniehobson** [comment]
The failing test is just a result of the slug changes, I'll fix that today.

File: bedrock/firefox/views.py [hunk 1]
```diff
@@ -547,14 +547,23 @@ def download_thanks(request):
...
+        elif experience == 'aus-gruenden':
+            template = 'firefox/new/berlin/scene2-aus-gruenden.html'
+        elif experience == 'herz':
+            template = 'firefox/new/berlin/scene2-herz.html'
+        elif experience == 'geschwindigkeit':
+            template = 'firefox/new/berlin/scene2-gesch.html'
+        elif experience == 'privatsphare':
+            template = 'firefox/new/berlin/scene2-privat.html'
+        elif experience == 'auf-deiner-seite':
+            template = 'firefox/new/berlin/scene2-ads.html'
```
Comment (alexgibson) [inline bedrock/firefox/views.py]: Should this be `scene2-auf-deiner-seite.html`?

File: bedrock/firefox/templates/firefox/new/berlin/scene2-auf-deiner-seite.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/new/quantum" %}
+
+{% extends "firefox/new/berlin/base-variation.html" %}
+
+{% block body_class %}berlin-scene2 ads{% endblock %}
+
+{% block page_og_url %}{{ settings.CANONICAL_URL }}/{{ LANG }}/firefox/new/?xv=herz{% endblock %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/new/berlin/scene2-auf-deiner-seite.html]: Do the slugs on these canonical URLs need updating?

File: media/css/firefox/new/berlin-variation.scss [hunk 3]
```diff
@@ -101,15 +130,39 @@ a.button:visited.button-green:focus {
...
+                min-height: 0;
+            }
+        }
+
         background-position: top -20px center;
     }
 
     @media #{$mq-desktop} {
-        background: #23033d url('/media/img/firefox/new/berlin/mit-herz.jpg') top center no-repeat;
+        background: #23033d url('/media/img/firefox/new/berlin/mit-herz.png') top center no-repeat;
```
Comment (alexgibson) [inline media/css/firefox/new/berlin-variation.scss]: 👍

**Comment by stephaniehobson** [comment]
Updated 🚀

**Pull request merged**